CONTRIBUTING.md

```diff
@@ -1,227 +1,14 @@
-    resolving the JBS issue. You do this with the "Resolve" action in JBS,
-    selecting "Fixed" as the resolution. You also need to add the commit
-    notification message (minus the list of modified files) as a comment.
-    This includes the URL of the commit. For example:
-
-    ```
-    Changeset: 1de25a49
-    Author:    Kevin Rushforth <kcr@openjdk.org>
-    Date:      2019-09-23 08:15:36 +7000
-    URL:       https://git.openjdk.java.net/jfx/commit/1de25a49
-
-    8231126: libxslt.md has incorrect version string
-
-    Reviewed-by: ghb
-    ```
-
-
-Contributing to the OpenJFX codebase
-------------------------------------------
-
-JDK 11 (at a minimum) is required to build OpenJFX. You must have a JDK 11 installation
-with the environment variable `JAVA_HOME` referencing the path to Java home for
-your JDK 11 installation. By default, tests use the same runtime as `JAVA_HOME`.
-Currently OpenJFX builds are running on JDK 11 (recommended) and JDK 12.
-
-It is possible to develop in any major Java IDE (Eclipse, IntelliJ, NetBeans). IDEs can automatically configure projects based on Gradle setup.
-
-The following formatting rules are enforced for source code files by
-`git jcheck`, which is run by the Skara bot:
-
-* Use Unix-style (LF) line endings not DOS-style (CRLF)
-* Do not use TAB characters (exception: Makefiles can have TABS)
-* No trailing spaces
-* No files with execute permission
-
-Please also follow these formatting guidelines:
-
-* Java indent is 4 spaces
-* Line width is no more than 120 characters
-* The rest is left to Java coding standards
-* Disable &ldquo;auto-format on save&rdquo; to prevent unnecessary format changes. This makes reviews much harder as it generates unnecessary formatting changes. If your IDE supports formatting only modified chunks that is fine to do.
-* Wildcard imports (`import foo.bar.baz.*`) are forbidden and may cause the build to fail. Please attempt to tame your IDE so it doesn't make them and please send a PR against this document with instructions for your IDE if it doesn't contain them.
-* Don't worry too much about import order. Try not to change it but don't worry about fighting your IDE to stop it from doing so.
-
-OpenJFX uses Gradle for its build. Before submitting your changes, run the test suite to make sure that nothing is broken, with:
-
-```sh
-bash ./gradlew all test
 ```
-
-If you are changing anything that might possibly affect rendering, you should run a full test with robot enabled:
-
-```sh
-bash ./gradlew -PFULL_TEST=true -PUSE_ROBOT=true all test
+Backport LONG-COMMIT-HASH
 ```
 
-Even more documentation on OpenJFX projects and its build system can be found on the
-[OpenJFX Wiki](https://wiki.openjdk.java.net/display/OpenJFX/).
+where `LONG-COMMIT-HASH` is the long (40 char) commit hash of the fix as found in the main jfx repo. The Skara tooling will then note that it is a backport, and replace the title with the correct issue title.
```

Comment:
Should the next parts of the process be explained as well (e.g. optional review if no clean patch, merging?)

Reply:
Good idea, but I'd prefer to do that as a follow-on fix; that change could then be backported to jfx11u (as a clean backport).

Reply:
makes sense

Reply:
I filed https://bugs.openjdk.java.net/browse/JDK-8273732 as a follow-up.